packageManifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "packageName": "Google Home Community",
   "author": "Miles Budnek",
-  "version": "0.27.0",
+  "version": "0.27.1",

```

Comment:
A new trait is probably worthy of a minor version bump to `"0.28.0"`.

Reply:
Updated.